roles/installer/vars/main.yml

```diff
@@ -8,3 +8,16 @@ supported_pg_version: 15
 _previous_upgraded_pg_version: 0
 old_postgres_pod: []
 _postgres_data_path: '/var/lib/pgsql/data/userdata'
+# metrics-utility (github.com/ansible/metrics-utility)
+_metrics_utility_enabled: "{{ metrics_utility_enabled | default(false) }}"
+_metrics_utility_configmap: "{{ metrics_utility_configmap | default(deployment_type + '-metrics-utility-configmap') }}"
+_metrics_utility_secret: "{{ metrics_utility_secret | default('') }}"
```

Comment:
Critically, this now defaults to an empty string instead of a generic secret name that will not exist.